### DISCUSSION TITLE
reconcile account

### DISCUSSION THREAD

**Discussion by Bilis518** [pspete/psPAS#510] [Q&A] [answered]
hi everyone,
on the add-pasAccount command, i see in the help arguments ExtraPass3... are used with gen1 api. How to specify these arguments using gen2 api?
also, how to read these values when using get-pasAccount?
thanks for help.
regards

**Comment by pspete** [comment]
https://pspas.pspete.dev/commands/Set-PASLinkedAccount

**Reply by Bilis518** [reply]
thanks for this answer, api is version 11.x 😭...

**Reply by pspete** [reply, marked as answer]
Answered here:  https://github.com/pspete/psPAS/issues/314

https://cyberark.my.site.com/s/article/Add-Reconcile-and-Login-Accounts-to-an-Account-using-V10-REST-API

**Comment by Bilis518** [comment]
many thanks for your quick response and great work.
